qubes/storage/reflink.py

```diff
@@ -438,7 +438,7 @@ def _copy_file(src, dst):
                 raise qubes.storage.StoragePoolException(str(result))
     return reflinked
 
-def is_supported(dst_dir, src_dir=None):
+def is_supported(dst_dir, *, src_dir=None):
```

Comment:
Should both arguments be keyword-only?  That would be even easier to read at the expense of some verbosity and a bigger diff.  Obviously this is optional.

Reply:
I'm sad to report that I had bikeshedded these alternatives (and a third: deleting `src_dir`) in my own head for a silly amount of time. This one seemed fine I guess lol